Project.toml

```
@@ -12,10 +12,10 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 DataFrames = "1"
-DocStringExtensions = "0.8.5, 0.9"
+DocStringExtensions = "0.8.5, ~0.9"
```

isn't it the same thing, to specify `~0.9` and `0.9`?
https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0

---

i think it is, so i'll put this back for sake of simplicity
```suggestion
DocStringExtensions = "0.8.5, 0.9"
```